discohash_512.cpp

```diff
@@ -0,0 +1,172 @@
+// also known as BEBB4185
```

Comment:
Really? This not I assume. Only the one with the 256bit state

Reply:
Not! 😅 Thanks :)